source/Nice3point.Revit.Toolkit/Helpers/ResolveHelper.cs

```diff
@@ -51,6 +51,12 @@ public static void BeginAssemblyResolve<T>()
     public static void BeginAssemblyResolve(Type type)
     {
         if (_domainResolvers is not null) return;
+        
+        // https://learn.microsoft.com/en-us/dotnet/api/system.reflection.module.fullyqualifiedname?view=netframework-4.8
+        if (type.Module.FullyQualifiedName.Equals("<Unknown>"))
+        {
+            throw new NotSupportedException("Type was load by raw assembly.");
```

Comment:
I guess the best approach is to use `return` instead of throwing an exception that this case was handled by other resolvers @dosymep

Reply:
done